README.md

```diff
@@ -129,6 +129,11 @@ Dependent repositories:product-ei
 Repository URL <https://github.com/wso2/msf4j>  
 Microservices implementation framework for java
 
+## Connectors
+
+WSO2 EI supports connectors available at the repository 
+Repository URL <https://github.com/wso2-extensions/?utf8=%E2%9C%93&q=&type=&language=>
```

Comment:
Since there are other identity and siddhi extensions as well in the extensions repo, shall we update the url as https://github.com/wso2-extensions?utf8=%E2%9C%93&q=esb&type=&language= to filter the esb-extensions.

Reply:
Thanks. I will close this PR and address this in a new one